Dockerfile

```diff
@@ -15,22 +15,24 @@ ARG BASE_IMAGE_NAME=debian:buster-slim
 # Intermediate build container
 FROM $BASE_IMAGE_NAME AS builder
 
-ENV PYTHON_VERSION 3.9.6
-ENV PYTHON_PIP_VERSION 21.1.3
+# Set Python and Pip versions
+ENV PYTHON_VERSION=3.9.6 \
```

Comment:
This is not necessary (as well as the other changes in 9431c4a0efd10eced93b2ffaf11d1c0accfabb6d). As already explained in https://github.com/eht16/python3-docker/pull/1#discussion_r666512786: this is a temporary build container and all its layers will be dropped and will not be included in the final image.